app/views/decidim/challenges/challenges/show.html.erb

```diff
@@ -0,0 +1,171 @@
+          <%= present(@challenge).coordinating_entities %>
+        </div>
+
+        <div class="definition-data__item col_entity">
+          <span class="definition-data__title"><%= t("col_entity", scope: "activemodel.attributes.challenge") %></span>
+          <%= present(@challenge).collaborating_entities %>
+        </div>
+
+        <div class="definition-data__item sdg">
+          <span class="definition-data__title"><%= t("sdg", scope: "activemodel.attributes.challenge") %></span>
+          <%= sdgs[present(@challenge).sdg] %><br/>
+          <div class="ods">
+              <div class="text-container show">
+                  <p><%= t("no_poverty.logo.line1", scope: "decidim.components.sdgs") %></p>
+                  <p><%= t("no_poverty.logo.line2", scope: "decidim.components.sdgs") %></p>
+              </div>
+              <img alt="Logo ODS <%= (present(@challenge).sdg + 1).to_s.rjust(2, '0') %>"
+                src="/assets/decidim/sdgs/ods-<%= (present(@challenge).sdg + 1).to_s.rjust(2, '0') %>.svg">
+          </div>
+        </div>
+        </div>
+
+      <%= resource_reference(current_participatory_space) %>
+      <%= render partial: "decidim/shared/share_modal" %>
+      <!-- embed_modal_for participatory_process_participatory_process_widget_url(current_participatory_space, format: :js)  -->
+
+  </div>
+
+  <div class="columns mediumlarge-8 mediumlarge-pull-4">
+    <button class="button secondary hollow small has-tip">
+      < Tornar al llistat </button> <br />
+
+      <div class="section">
+        <h2 class="heading2"><strong><%= present(@challenge).title %></strong></h2><br />
+
+        <h3><strong><%= t("local_description", scope: "activemodel.attributes.challenge") %></strong></h3>
+        <p><%= present(@challenge).local_description %></p>
+
+        <h3><strong><%= t("global_description", scope: "activemodel.attributes.challenge") %></strong></h3>
+        <p><%= present(@challenge).global_description %></p>
+
+        <h3><strong><%= t("tags", scope: "activemodel.attributes.challenge") %></strong></h3>
+        <p class="tags"><%= present(@challenge).tags %></p>
+
+        <h3><strong>Problemes associats</strong></h3>
+
+        <div class="row small-up-1 medium-up-2 card-grid">
+
+          <div class="column">
+            <article class="card">
+              <div class="card__content">
+                <div class="card__header">
+                  <a class="card__link" href="/processes/PDUAECOConcaOdena/f/2843/debates/31">
+                    <h5 class="card__title">
+                      <strong>Problema #1</strong>
+                    </h5>
+                  </a>
+                </div>
+                <div class="card__text">
+                  <div class="card__text--paragraph">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse in ex a neque pellentesque
```

Comment:
Be aware with these "Lorem ipsum..." texts over there... They shouldn't be there please..